### DISCUSSION TITLE
General uses and type of users of GeoClimate

### DISCUSSION THREAD

**Discussion by j3r3m1** [orbisgis/geoclimate#936] [Community]
It is always difficult to identify who are the GeoClimate users and what are the GeoClimate uses, thus making difficult for the developers to show to potential sponsors what kind of community they are funding. 

In this thread is expected for each user few (optional)  informations:

1. what kind of organization GeoClimate users belong to (e.g. research institute, university, study offices, local authority, etc.),
2. what is the name of the organization ?
3. do you use GeoClimate for commercial purpose ?
4. what is the purpose of using GeoClimate : urban climate modelling, building energy, urban planning, etc.
5. what is the scale of your analysis (neighborhood scale, commune scale, department, region, etc.) ?
6. how frequently do you run GeoClimate ?
7. what are the GeoClimate data you are interested in ?
    a. input data (such as building, road, vegetation, etc.)
    b. output data (building indicators, rsu indicators, rsu lcz, grid indicators, etc.)

**Comment by j3r3m1** [comment]
Here is an example:
1. Research Institute
2. CNRM (France)
3. No
4. Urban climate modelling (using physical and statistical methods)
5. Commune or conurbation scale but also neighborhood scale for some uses
6. Every week (but mainly for tests...)
7a. buildings
7b. all output tables

**Comment by atoulou** [comment]
Hello all,
I'm using Geoclimate for my PhD research, here are the details:
1. Research Institute
2. CETHIL (Centre d'Energétique et de Thermique de Lyon) UMR 5008, France
3. No
4. Urban climate modelling (classification of a territory according to its climatic behavior)
5. Neighborhood scale
6. Once, but potentially other times if new zones are required 
7. a. all but above all buildings / vegetation 
7. b. all but mainly LCZ

**Comment by yoannlmr** [comment]
Hi, everyone !

1. Research Institute and Study Office
2. Université Paris-Saclay and Hyperion (France)
3. No yet
4. Territorial resilience modelling
5. Commune scale
6. First time last week, I hope once a month
7 a. All inputs data
7 b. Mainly RSU indicators, RSU LCZ

Thank's, have fun !

**Comment by Denis-Vannier** [comment]
Hi !

I'm an independant data-journalist, founder of "Le Plan Studio", working on climate issues in french cities. 

1 - Data-journalism studio
2 - "Le Plan Studio"
3 - I will sell journalism papers or datavisualisations that may use GeoClimate as source.
4 - urban climate modelling
5 - neighborhood scale
6 - I run it twice for the moment, but I plan to use it for 30 cities.
7 - all output data

Thank you so much !
